Formula/graph-tool.rb

```diff
@@ -0,0 +1,90 @@
+class GraphTool < Formula
+  include Language::Python::Virtualenv
+
+  desc "Efficient network analysis for Python 3"
+  homepage "https://graph-tool.skewed.de/"
+  url "https://downloads.skewed.de/graph-tool/graph-tool-2.26.tar.bz2"
+  sha256 "df6273dc5ef327a0eaf1ef1c46751fce4c0b7573880944e544287b85a068f770"
+
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "boost-python3"
+  depends_on "cairomm"
+  depends_on "cgal"
+  depends_on "google-sparsehash"
+  depends_on "gtk+3"
+  depends_on :macos => :el_capitan # needs thread-local storage
+  depends_on "librsvg"
+  depends_on "numpy"
+  depends_on "py3cairo"
+  depends_on "pygobject3"
+  depends_on "python"
+  depends_on "scipy"
+
+  resource "Cycler" do
+    url "https://files.pythonhosted.org/packages/c2/4b/137dea450d6e1e3d474e1d873cd1d4f7d3beed7e0dc973b06e8e10d32488/cycler-0.10.0.tar.gz"
+    sha256 "cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8"
+  end
+
+  resource "matplotlib" do
+    url "https://files.pythonhosted.org/packages/50/27/57ab73d1b094540dec1a01d2207613248d8106f3c3f40e8d86f02eb8d18b/matplotlib-2.1.1.tar.gz"
+    sha256 "659f5e1aa0e0f01488c61eff47560c43b8be511c6a29293d7f3896ae17bd8b23"
+  end
+
+  resource "pyparsing" do
+    url "https://files.pythonhosted.org/packages/3c/ec/a94f8cf7274ea60b5413df054f82a8980523efd712ec55a59e7c3357cf7c/pyparsing-2.2.0.tar.gz"
+    sha256 "0832bcf47acd283788593e7a0f542407bd9550a55a8a8435214a1960e04bcb04"
+  end
+
+  resource "python-dateutil" do
+    url "https://files.pythonhosted.org/packages/54/bb/f1db86504f7a49e1d9b9301531181b00a1c7325dc85a29160ee3eaa73a54/python-dateutil-2.6.1.tar.gz"
+    sha256 "891c38b2a02f5bb1be3e4793866c8df49c7d19baabf9c1bad62547e0b4866aca"
+  end
+
+  resource "pytz" do
+    url "https://files.pythonhosted.org/packages/60/88/d3152c234da4b2a1f7a989f89609ea488225eaea015bc16fbde2b3fdfefa/pytz-2017.3.zip"
+    sha256 "fae4cffc040921b8a2d60c6cf0b5d662c1190fe54d718271db4eb17d44a185b7"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/16/d8/bc6316cf98419719bd59c91742194c111b6f2e85abac88e496adefaf7afe/six-1.11.0.tar.gz"
+    sha256 "70e8a77beed4562e7f14fe23a786b54f6296e34344c23bc42f07b15018ff98e9"
+  end
+
+  def install
+    xy = Language::Python.major_minor_version "python3"
+
+    inreplace "configure", "libboost_python", "libboost_python36"
```

Comment:
Is there some way we can pass this and the line below to configure? Upstream needs to be able to handle the Boost 1.67.0 Python libraries.